src/main/java/appeng/me/storage/SecurityStationInventory.java

```diff
@@ -51,7 +51,7 @@ public IAEItemStack injectItems( final IAEItemStack input, final Actionable type
 	{
 		if( this.hasPermission( src ) )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2583)